components/bl-transitions-component/component.json

```
@@ -36,23 +36,26 @@
           "label": "Collapse Left"
         }
       ],
-      "hasLogicHandler": false
+      "hasLogicHandler": false,
+      "settingTooltip": "Allows to determine variant of transition"
```

```suggestion
      "settingTooltip": "Allows to choose a variant of transition"
```

Please google the difference between choose and determine